### PR TITLE
island-ui/Update the Tag component

### PR DESCRIPTION
**Pull request by oddsson**
# Update the Tag component

## What

Remove prop `label` and `bordered` and add a prop `outlined` to the Tag component.

## Why

According to the design, the tag component should be able to be filled or outlined. The filled version is clickable while the outlined is not. Also, according to the desing, we should not offer filled and bordered tags.

## Screenshots / Gifs

Screen Shot 2020-11-26 at 11 38 36

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by addi — DISMISSED** [review]

**Review by atligudl — APPROVED** [review]
Service Portal looks good

**Review by atligudl — APPROVED** [review]

**Review by barabrian — APPROVED** [review]
LGTM

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Review by dd-sdang — APPROVED** [review]

File: libs/island-ui/core/src/lib/Tag/Tag.tsx [hunk 1]
```diff
@@ -70,7 +67,7 @@ export const Tag = forwardRef<HTMLButtonElement & HTMLAnchorElement, TagProps>(
       </Text>
     )
 
-    if (label) {
+    if (outlined) {
```
Comment (barabrian) [inline libs/island-ui/core/src/lib/Tag/Tag.tsx:70]: Just a though for another PR.. I'm not sure we want to define clickable or not by the outlined value. Would it make sense to use a button if the `onClick` function is present? something like what's happening with the `href` attribute? so if there is no `href` and no `onClick` then we would return this span?

**Review by addi — APPROVED** [review]

**Pull request merged**